lansa-vmss-windows-autoscale-sql-database/mainTemplate.json

```diff
@@ -676,7 +753,7 @@
                "osProfile": {
                   "computerNamePrefix": "[variables('dbvmssName')]",
                   "adminUsername": "[parameters('adminUsername')]",
-                  "adminPassword": "[parameters('adminPassword')]",
+                  "vaultSecretID": "[parameters('vaultSecretID')]",
```

Comment:
Please explain this. Key Vault is nothing to do with the VM

Reply:
Changed "adminPassword" with "vaultSecretId" by mistake here. I have reverted this change.